src/locutus/api/table_mappings.py

```diff
@@ -91,10 +91,15 @@ def get(self, id, code):
             editor = get_editor(body=None, editor=editor_param)
             if user_input_param is not None and editor is None:
                 raise LackingUserID(editor)
-            
+
             table = Table.get(id)
             term = table.terminology.dereference()
 
+            # Ensure codes are not placeholders at this point.
+            code = (
+                normalize_ftd_placeholders(code) if code in FTD_PLACEHOLDERS else code

```

Comment:
If you change the function, normalize_ftd_placeholders to return code if there is nothing to replace, you can drop that if else you are doing here.